frontend/src/components/plugins/utils/utils.jsx

```diff
@@ -32,10 +32,18 @@ import {
   CONNECTORS_CONFIG_URI,
   VISUALIZERS_CONFIG_URI,
 } from "../../../constants/api";
-import { pluginType } from "../../../constants/constants";
+import { pluginType, scanMode } from "../../../constants/constants";
 
 const { checkPluginHealth } = usePluginConfigurationStore.getState();
 
+function parseScanCheckTime(time) {
+  // scan_check_time is in format days:hours:minutes:seconds, we need to convert them to hours
+  const [daysAgo, hoursAgo] = time
+    .split(":")
+    .map((token) => parseInt(token, 10));
+  return daysAgo * 24 + hoursAgo;
+}
+
```

Comment:
This is the same logic used in ScanForm:
```
...
if (scanCheckTime) {
      // scan_check_time is in format days:hours:minutes:seconds, we need to convert them to hours
      const [daysAgo, hoursAgo] = scanCheckTime
        .split(":")
        .map((token) => parseInt(token, 10));
      formik.setFieldValue("scan_check_time", daysAgo * 24 + hoursAgo, false);
    }
```

export this function and use this in scanForm, in this way we don't have duplicated code

Reply:
After exporting this function add a specific unittest